src/app/productDetail/templates/productDetail.html

```diff
@@ -5,12 +5,16 @@
     <hr>
     <div class="row">
         <div class="col-md-6">
-            <!--PRODUCT IMAGE-->
-            <div class="thumbnail">
+            <!-- PRODUCT IMAGES -->            
+            <div class="thumbnail" style="max-width: 400px; height: auto; margin: 0 auto;" ng-if="!productDetail.item.xp.additionalImages || !productDetail.item.xp.additionalImages.length">
```

Comment:
@amrarick26 would you like to update this PR with the changes we talked about?  Or just submit a new one once you have time to do it.

Basically, display a single image with a lightbox if there aren't any additional images and image zoom is turned on.